src/main/java/Launcher.java

```diff
@@ -0,0 +1,10 @@
+import javafx.application.Application;
```

Comment:
Perhaps you should group this class in a package? Might help reviews understand the code better!